src/proj.rs

```diff
@@ -1086,13 +1089,15 @@ mod test {
         let from = "EPSG:2230";
         let to = "EPSG:26946";
         let ft_to_m = Proj::new_known_crs(&from, &to, None).unwrap();
-        let mut v = vec![
-            MyPoint::new(4760096.421921, 3744293.729449),
-            MyPoint::new(4760197.421921, 3744394.729449),
-        ];
-        ft_to_m.convert_array(&mut v).unwrap();
-        assert_almost_eq(v[0].x(), 1450880.2910605003f64);
-        assert_almost_eq(v[1].y(), 1141293.7960220212f64);
+        for _ in 0..100 {
```

Comment:
I understand the motivation for this change based on the context we're in right now, but eventually it'll be weird and inexplicable. I'd say either revert the loop change or pull it into a separate test with an explanation of its provenance.